pkg/image/external/external.go

```diff
@@ -8,9 +8,11 @@ import (
 	"strings"
 
 	"github.com/coreos/go-semver/semver"
+	"github.com/sirupsen/logrus"
+
```

Comment:
there should not be a space between these imports, please remove it and perform a goimports to re-sort (or manually sort) the imports.